File: GroupExpenses.Domain/Entities/Receipt.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace GroupExpenses.Domain.Entities
{
   public class Receipt
   {
         public int Id { get; set; }
         public int EventId { get; set; }
         [ForeignKey(nameof(EventId))]
         public Event Event { get; set; }
         public string Name { get; set; }
         public decimal Price { get; set; }
         public decimal PriceInEur { get; set; }
         public string Details { get; set; }
         public int Currency { get; set; }
         public User PaidBy { get; set; }
         public IEnumerable<User> PaidFor { get; set; }
   }
}

File: GroupExpensesMobile/src/types/receipt.ts
import {Participant} from './event';

export interface Receipt {
  id: number;
  eventId: number;
  name: string;
  price: number;
  priceInEur: number;
  description: string | null;
  currency: number | string;
  paidBy: Participant;
  paidFor: Participant[];
}
